src/main/java/net/tazpvp/tazpvp/utils/objects/bosses/zorg/attacks/SonicBoomAttack.java

```diff
@@ -17,7 +15,7 @@ public class SonicBoomAttack implements Attack {
     private final Random random = new Random();
 
     @Override
-    public void attack(BOSSSSS boss) {
+    public void attack(CustomBoss boss) {
         final RayTraceResult result = boss.getBoss().getWorld().rayTraceEntities(
                 ((WitherSkeleton) boss.getBoss()).getEyeLocation().add(boss.getBoss().getLocation().getDirection()),
                 ((WitherSkeleton) boss.getBoss()).getEyeLocation().getDirection(),
```

Comment:
Make a variable for casting entity to wither skeleton, rather than casting it twice.